src/pages/resolve/components/IpfsLink.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <a :href="hashLink" class="hash-link" target="_blank">{{ hash }}</a>
+</template>
+
+<script>
+export default {
+  props: ["hash"],
+  computed: {
+    hashLink() {
+      return `https://api.dstor.cloud/ipfs/${this.hash}`;
+    },
+  },
+};
+</script>
+
+<style scoped>
```

Comment:
Verify your styles don't break anything outside this scope.

```suggestion
<style>
```